### ISSUE TITLE
Too few arguments, still happening

### ISSUE THREAD

**Issue by MethylBromide** [thomasgriffin/TGM-Plugin-Activation#799]
Warning: sprintf(): Too few arguments in …/class-bulk-upgrader-skin.php on line 152

Fix is described here: https://gist.github.com/al5dy/965d0eefffab32df3b04badbc2274d08

Please make the indicated code change on line 24 of class-tgm-plugin-activation.php -- the string skin_update_successful is used in a context where sprintf is called with one data argument, and the formatting string you are assigning currently has two placeholders.

**Comment by RupW** [comment]
This probably best as a pull request, if you can make one please?

**Comment by jrfnl** [comment]
This has already been fixed in the code here. There are a [number of other things](https://github.com/TGMPA/TGM-Plugin-Activation/issues/741#issuecomment-488142682) though which a release is waiting on, pull requests for those are very welcome.

**Comment by jrfnl** [comment]
Marking this as a duplicate of #669 which was fixed by #670.
